vxyowsup/whatsapp.py

```diff
@@ -124,8 +126,11 @@ def __init__(self, transport):
         super(WhatsAppInterface, self).__init__()
         self.transport = transport
 
-    def send_to_human(self, text, to_address):
+    def send_to_human(self, text, to_address, message_id):
+        # message_id is vumi id
         message = TextMessageProtocolEntity(text, to=to_address)
+        self.transport.redis.setex(message.getId(), 60 * 60 * 24, message_id)
```

Comment:
It is polite to either put constants like `60 * 60 * 24` on the class, e.g.
```python
def MyTransport(self):

    MESSAGE_ACK_TIMEOUT = 60 * 60 * 24
```
or to make them configurable (bonus points).